starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,57 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+
+.text {
+    font-size: 11px;
+    font-weight: 200;
+    margin-left: 29%;
+    margin-right: 29%;
+    font-family: Arial;
+    margin-bottom: 20px;
+}
+
+h1 {      
+    font-size: 60px;
+    text-align: center;
+    color: white;
+
+}
+
+h2 {
+    font-size: 16px;
+    line-height: 4em;
+    
+}
+
+h3 {
+    color: black;
+    font-size: 14px;
+    text-align: center;
+    padding-top: 50px;
```

Comment:
You coud set the font-weight to 300 so the text appears less thick.
font-weight:300;

https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight